python/example_code/s3/ReadMe.md

```diff
@@ -0,0 +1,80 @@
+# Amazon Simple Storage Service (S3) API examples
+
+## Purpose
+
+Demonstrate basic bucket and object operations in Amazon Simple Storage Service (S3).
+Learn how to create, get, remove, and configure buckets and objects.
+
+## Prerequisites
+
+- You must have an AWS account, and have your default credentials and AWS Region
+  configured as described in the [AWS Tools and SDKs Shared Configuration and
+  Credentials Reference Guide](https://docs.aws.amazon.com/credref/latest/refdocs/creds-config-files.html).
+- Python 3.6 or later
+- Boto 3 1.11.10 or later
+- PyTest 5.3.5 or later (to run unit tests)
+
+## Running the code
+
+Run individual functions in the Python shell to make requests to your AWS account.
+For example, run the following commands to create a bucket, upload an object, get
+the object, empty the bucket, and delete the bucket.  
+
+    > python
+    >>> import time
+    >>> bucket_name = f"bucket{time.time_ns()}"
+    >>> import bucket_wrapper
+    >>> bucket = bucket_wrapper.create_bucket(bucket_name, region='us-west-2')
+    >>> import object_wrapper
+    >>> object_wrapper.put_object(bucket, 'my-test-object', b'My test data')
+    >>> obj = object_wrapper.get_object(bucket, 'my-test-object')
+    >>> object_wrapper.empty_bucket(bucket)
+    >>> bucket_wrapper.delete_bucket(bucket)
+
+## Running the tests
+
+The best way to learn how to use this service is to run the tests.
+Tests can be run in two modes. By default, tests use the Botocore Stubber,
+which captures requests before they are sent to AWS and returns a mocked response.
+Tests can also be run against your AWS account, in which case they will create and 
+manipulate AWS resources, which may incur charges on your account.
+
+To run all of the S3 tests with the Botocore Stubber, run the following in
+your [GitHub root]/python/example_code/s3 folder.
+
+    python -m pytest -o log_cli=1 --log-cli-level=INFO test
+
+The '-o log_cli=1 --log-cli-level=INFO' flags configure pytest to output
+logs to stdout during the test run. Without them, pytest captures logs and prints
+them only when the test fails.
+
+To run the tests using your AWS account and default shared credentials, include the
+'--use-real-aws-may-incur-charges' flag.
+
+    python -m pytest -o log_cli=1 --log-cli-level=INFO --use-real-aws-may-incur-charges test
+
+When run in this mode, a best effort is made to clean up any resources created during 
+the test, but it is your responsibility to verify that all resources have, in fact, 
+been cleaned up.
+
+## Additional resources
+
+- Boto 3, see the [Boto 3 SQS documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/sqs.html).
```

Comment:
Does this need to be a reference to S3 guide instead?

Reply:
Yes, it does!